### DISCUSSION TITLE
Relay control

### DISCUSSION THREAD

**Discussion by andremalimpensa** [awawa-dev/HyperSerialEsp8266#28] [General]
Hey, congrats for the excellent work!
Please, would it possible to use one of the GPIOs to control a 5v relay? Setting it to high only when the serial port is connected or something along those lines?
Thank you.

**Comment by awawa-dev** [comment]
Hi
You can hook your GPIO routines around these lines:
active
https://github.com/awawa-dev/HyperSerialEsp8266/blob/6c7251ed7c652b061e64e9334e2d41ecd638db6e/include/main.h#L67
not active
https://github.com/awawa-dev/HyperSerialEsp8266/blob/6c7251ed7c652b061e64e9334e2d41ecd638db6e/include/main.h#L71
